src/pytest_regressions/common.py

```diff
@@ -69,42 +80,42 @@ def check_text_files(obtained_fn, expected_fn, fix_callback=lambda x: x, encodin
 
 
 def perform_regression_check(
-    datadir,
-    original_datadir,
-    request,
-    check_fn,
-    dump_fn,
-    extension,
-    basename=None,
-    fullpath=None,
-    force_regen=False,
-    with_test_class_names=False,
-    obtained_filename=None,
-    dump_aux_fn=lambda filename: [],
-):
+    datadir: Path,
+    original_datadir: Path,
+    request: pytest.FixtureRequest,
+    check_fn: Callable[[Path, Path], None],
```

Comment:
Oh... not related to the changes, but I've just noticed that we're using `fn` to denote `function` here, and we're using `fn` some lines above to denote `filename` (and this is already etched in the API) 😓